typings/image-capture.d.ts

```diff
@@ -140,12 +131,7 @@ interface ConstrainPoint2DParameters {
 
 declare type ConstrainPoint2D = Point2D[] | ConstrainPoint2DParameters;
```

Comment:
This is copy and paste straight from the spec: https://w3c.github.io/mediacapture-image/#typedefdef-constrainpoint2d